src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -831,6 +856,41 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
         this.actionOnItemEvent(event.data, mapContributor, resultListContributor, currentCollection);
         break;
       case 'globalActionEvent':
+        if (event.data.id === 'production') {
+          const idsItemSelected: ElementIdentifier[] = this.featuresToSelect;
+          this.processService.load().subscribe({
+            next: () => {
+
+              this.processService.getItemsDetail(
+                this.collectionToDescription.get(currentCollection).id_path,
+                idsItemSelected.map(i => i.idValue),
+                this.processService.getProcessDescription().additionalParameters?.parameters,
+                currentCollection
+              ).subscribe({
+                next: (item: any) => {
+                  this.downloadDialogRef = this.dialog.open(ProcessComponent, { minWidth: '520px', maxWidth: '60vw' });
+                  this.downloadDialogRef.componentInstance.nbProducts = this.featuresToSelect.length;
+                  this.downloadDialogRef.componentInstance.matchingAdditionalParams = item as Map<string, boolean>;
+                  this.downloadDialogRef.componentInstance.wktAoi = this.mapglComponent.getAllPolygon('wkt');
+                  this.downloadDialogRef.afterClosed().subscribe({
+                    next: (data) => {
+                      if (!!data) {
+                        this.processService.process(
+                          idsItemSelected.map(i => i.idValue),
+                          data.payload,
+                          currentCollection
+                        ).subscribe({
+                          next: (result) => console.log(result)
```

Comment:
It could be interesting to notify the user through the interface that their request succeeded